kinde_flask/framework/flask_framework.py

```
@@ -29,13 +32,18 @@ def __init__(self, app: Optional[Flask] = None):
         self._initialized = False
         self._oauth = None
         
-        # Configure Flask session
+        # Configure Flask session for server-side storage
+        # This is required because OAuth tokens can exceed cookie size limits (~4KB)
         self.app.config['SECRET_KEY'] = os.getenv('SECRET_KEY', 'your-secret-key')
-        self.app.config['SESSION_TYPE'] = 'filesystem'
+        self.app.config['SESSION_TYPE'] = os.getenv('SESSION_TYPE', 'filesystem')
         self.app.config['SESSION_PERMANENT'] = False
+        self.app.config['SESSION_FILE_DIR'] = os.getenv('SESSION_FILE_DIR', '/tmp/flask_sessions')
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/flask_sessions"

(S108)